### PR TITLE
installer: Cleanup OKD config

### PR DESCRIPTION
**Pull request by LorbusChris**
- use base image from origin CI registry namespace on release-4.9 branch
- remove unused OCP CLI image from configs
- skip tests for docs and OWNERS file changes
- refer to OKD CLI image as origin_cli instead of misleading origin_4.8_cli on release-4.{8,9,10} branches

### CODE REVIEW

File: ci-operator/config/openshift/installer/openshift-installer-master__okd.yaml [hunk 1]
```diff
@@ -187,13 +185,14 @@ tests:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp
 - as: e2e-aws-upgrade
+  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(azure|baremetal|gcp|kubevirt|libvirt|openstack|ovirt|vsphere)/)
   steps:
     cluster_profile: aws
-    workflow: openshift-upgrade-aws
+    workflow: openshift-upgrade-aws-latestinstaller
```
Comment (LorbusChris) [inline ci-operator/config/openshift/installer/openshift-installer-master__okd.yaml:191]: I don't really know if it's right to use the `-latestinstaller` workflow for OKD. Anyone know some context?

File: ci-operator/config/openshift/installer/openshift-installer-master__okd.yaml [hunk 2]
```diff
@@ -20,18 +16,15 @@ base_images:
...
     tag: rhel-8-golang-1.16-openshift-4.10
   origin_4.8_cli:
-    name: "4.9"
+    name: "4.10"
     namespace: origin
     tag: cli
   rhel-7:
     name: builder
     namespace: ocp
     tag: rhel-7-golang-1.16-openshift-4.9
```
Comment (elmiko) [inline ci-operator/config/openshift/installer/openshift-installer-master__okd.yaml:25]: should this be 4.10 as well?

File: ci-operator/config/openshift/installer/openshift-installer-release-4.10__okd.yaml [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+    namespace: ocp
+    tag: rhel-8-golang-1.16-openshift-4.10
+  origin_4.8_cli:
+    name: "4.10"
+    namespace: origin
+    tag: cli
+  rhel-7:
+    name: builder
+    namespace: ocp
+    tag: rhel-7-golang-1.16-openshift-4.9
```
Comment (elmiko) [inline ci-operator/config/openshift/installer/openshift-installer-release-4.10__okd.yaml:25]: same question, should this be 4.10?

**Review by elmiko — COMMENTED** [review]
looks mostly good to me, just a couple questions about the rhel7 images

File: core-services/image-mirroring/openshift/mapping_origin_4_10 [hunk 4]
```diff
@@ -129,6 +130,8 @@ registry.ci.openshift.org/origin/4.10:hyperkube quay.io/openshift/origin-hyperku
 registry.ci.openshift.org/origin/4.10:ibm-cloud-controller-manager quay.io/openshift/origin-ibm-cloud-controller-manager:4.10 quay.io/openshift/origin-ibm-cloud-controller-manager:4.10.0 quay.io/openshift/origin-ibm-cloud-controller-manager:latest
 registry.ci.openshift.org/origin/4.10:ibmcloud-machine-controllers quay.io/openshift/origin-ibmcloud-machine-controllers:4.10 quay.io/openshift/origin-ibmcloud-machine-controllers:4.10.0 quay.io/openshift/origin-ibmcloud-machine-controllers:latest
 registry.ci.openshift.org/origin/4.10:insights-operator quay.io/openshift/origin-insights-operator:4.10 quay.io/openshift/origin-insights-operator:4.10.0 quay.io/openshift/origin-insights-operator:latest
+registry.ci.openshift.org/origin/4.10:installer quay.io/openshift/origin-installer:4.10 quay.io/openshift/origin-installer:4.10.0 quay.io/openshift/origin-installer:latest
```
Comment (smarterclayton) [inline core-services/image-mirroring/openshift/mapping_origin_4_10:133]: Why was there no image for this before?  That surprised me a bit.

File: ci-operator/config/openshift/installer/openshift-installer-master__okd.yaml (hunk 2 above)
Comment (LorbusChris) [reply ci-operator/config/openshift/installer/openshift-installer-master__okd.yaml:25]: yes, thank you!

**Comment by LorbusChris** [comment]
repushed, so that all newly added tests on release branches are optional like on master

File: ci-operator/config/openshift/installer/openshift-installer-master__okd.yaml (hunk 1 above)
Comment (LorbusChris) [reply ci-operator/config/openshift/installer/openshift-installer-master__okd.yaml:188]: I looked at the code and this workflow takes the initial installer from a stable-initial imagestream which I don't think exists for OKD. Reverting this part.

**Pull request merged**